pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[build-system]
+requires = ["setuptools==61.0.0", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "deephaven-riptable"
+authors = [
+    {name = "Deephaven Data Labs", email = "python@deephaven.io"},
+]
+description = "The Deephaven/Riptable Integration Package"
+requires-python = ">=3.8"
+keywords = ["Deephaven", "Riptable", "Integration"]
+license = {text = "Apache 2.0"}
+classifiers = [
+    'Development Status :: 4 - Beta',
+    'Intended Audience :: Developers',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3 :: Only',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+]
+dependencies = [
+    "deephaven-core",
+]
+dynamic = ["version", "readme"]
+
+[tool.setuptools.dynamic]
+version = {attr = "deephaven_riptable.__version__"}
+readme = {file = ["README.md"]}
```

Comment:
Does this do a find and replace in README.md? In that case, shouldn't the README.md have a reference to version (instead of explicitly 0.1.0)?

Reply:
I don't think it does find/replace in the file. It simply replace the meta data 'readme' with the content of README.md.